src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -21,6 +21,278 @@ public class AttributeTests : WellKnownAttributesTestBase
     {
         #region Function Tests
 
+        [Fact]
+        [WorkItem(21194, "https://github.com/dotnet/roslyn/issues/21194")]
+        public void CircularTypeReferenceNotFound_WithIrrelevantType()
```

Comment:
>CircularTypeReferenceNotFound_WithIrrelevantType [](start = 20, length = 48)

I am not sure if name accurately reflects the scenario. I don't see any errors about type not found.

Reply:
What I mean is that it does not matter whether `C` exists or not. The attribute constructor with zero parameters will be picked regardless and no error. That's why I called it "irrelevant".

---
In reply to: [155961753](https://github.com/dotnet/roslyn/pull/23647#discussion_r155961753) [](ancestors = 155961753)

Reply:
Oh, nvm. I think you meant the "NotFound" portion. :-S

---
In reply to: [156189700](https://github.com/dotnet/roslyn/pull/23647#discussion_r156189700) [](ancestors = 156189700,155961753)